spec/models/lakeshore/ingest_spec.rb

```diff
@@ -205,4 +205,33 @@
       its(:represented_resources) { is_expected.to be_empty }
     end
   end
+
+  describe "#force_preferred_representation?" do
+    subject { ingest.force_preferred_representation? }
+
+    context "by default" do
+      let(:params) { {} }
+      it { is_expected.to be(false) }
+    end
+
+    context "when set to false" do
+      let(:params) { { metadata: { force_preferred_representation: "false" } } }
+      it { is_expected.to be(false) }
+    end
+
+    context "when set to true" do
+      let(:params) { { metadata: { force_preferred_representation: "true" } } }
+      it { is_expected.to be(true) }
+    end
+
+    context "when set to 1" do
+      let(:params) { { metadata: { force_preferred_representation: 1 } } }
```

Comment:
@RudyOnRails During a meeting with one of the citi devs awhile back, there was a question about this. IIRC, sometimes they use 0 or 1 for their booleans. I wasn't sure, so I was leaving it open-ended. @scossu does this matter? Otherwise, we could only accept a "true" string for the value of True.

Reply:
Leaving 0 or 1 is totally doable if the client chooses to communicate that way; my point is that the parameter checked inside the model is checking for `1` when the param clients will pass in will probably be converted to a string of `"1"`, since this is not an ActiveRecord attribute that is typecasted back to an integer.